src/Components/Articles/SingleArticle.jsx

```diff
@@ -3,14 +3,16 @@ import axios from "axios";
 import { useEffect, useState } from "react";
 import styles from "../../styles/SingleArticle.module.css";
 import CommentsSection from "../Comments/CommentsSection";
-
+import AddComment from "../Comments/AddComment";
 import Votes from "../Votes";
 
+
 const SingleArticle = () => {
 const [article, setArticle] = useState({})
 const [isLoading, setIsLoading] = useState(true);
 const {article_id} = useParams()
 const [errorMessage, setErrorMessage] = useState(undefined);
+// const [newComment, setNewComment] = useState("")
 
```

Comment:
I think its probably safe to remove these comments now, you dont want rouge comments floating about in your code :)